src/EncryptOnlyGnuPGCrypt.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Krak\Crypto;
+
+class EncryptOnlyPGCrypt implements Crypt
```

Comment:
OK, so change this name to EncryptOnlyCrypt and typehint the constructor as Crypt.